src/rules/prefer-in-document.js

```diff
@@ -3,6 +3,8 @@
  * @author Anton Niklasson
  */
 
+/*eslint complexity: ["error", {"max": 20}]*/
```

Comment:
This was the easiest way to fix the eslint issue. I believe the problem is in other parts of the code than the one I added.

Also `20` is the default in eslint, it looks like some other other preset this project is using set it to 15, so I think this is reasonable to put it back to `20` here.

Tell me what you think!